doc/src/manual/dates.md

```diff
@@ -559,7 +559,7 @@ julia> round(DateTime(2016, 8, 6, 20, 15), Dates.Day)
 Unlike the numeric [`round()`](@ref) method, which breaks ties toward the even number by default,
 the [`TimeType`](@ref)[`round()`](@ref) method uses the `RoundNearestTiesUp` rounding mode. (It's
 difficult to guess what breaking ties to nearest "even" [`TimeType`](@ref) would entail.) Further
-details on the available `RoundingMode` s can be found in the [API reference](https://docs.julialang.org/en/latest/stdlib/dates.html).
+details on the available `RoundingMode` s can be found in the [API reference](https://docs.julialang.org/en/stable/stdlib/dates/).
```

Comment:
If I'm not mistaken, I think this can be `[API reference](@ref Dates)`, which will tell Documenter to link to the Dates section.